src/lib/assets/styles/base.css

```diff
@@ -10,3 +10,4 @@
 @import 'icons.css';
 @import 'layout.css';
 @import 'modal.css';
+@import 'pikaday-custom.css';
```

Comment:
Publish imports this CSS also, should clean that up eventually: https://github.com/PRX/publish.prx.org/issues/491